packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -84,50 +84,49 @@ - (void)downloadManifestFromURL:(NSURL *)url
                                                      timeoutInterval:EXUpdatesDefaultTimeoutInterval];
   [self _setManifestHTTPHeaderFields:request];
   [self _downloadDataWithRequest:request successBlock:^(NSData *data, NSURLResponse *response) {
+      
+    NSHTTPURLResponse* httpResponse = (NSHTTPURLResponse *)response;
```

Comment:
i would add an `if ([response isKindOfClass:[NSHTTPURLResponse class])` check around this just to be extra safe

Reply:
Added it as an assert.